backend/composer/api/serializers.py

```diff
@@ -680,3 +680,18 @@ class Meta:
             "errors"
         )
 
+
+class KnowledgeStatementSerializer(ConnectivityStatementSerializer):
+    """Knowledge Statement"""
+    class Meta(ConnectivityStatementSerializer.Meta):
+        fields = (
```

Comment:
suggestion: I think we should add the reference_uri here since that's one of the fields we allow filtering on.